File: web-admin/src/main/java/com/isystk/sample/web/admin/controller/html/login/LoginForm.java
package com.isystk.sample.web.admin.controller.html.login;

import java.io.Serializable;
import javax.validation.constraints.NotEmpty;

public class LoginForm implements Serializable {

  private static final long serialVersionUID = 7593564324192730932L;

  @NotEmpty
  String loginId;

  @NotEmpty
  String password;

  public @NotEmpty String getLoginId() {
    return this.loginId;
  }

  public @NotEmpty String getPassword() {
    return this.password;
  }

  public void setLoginId(@NotEmpty String loginId) {
    this.loginId = loginId;
  }

  public void setPassword(@NotEmpty String password) {
    this.password = password;
  }
}

File: web-admin/src/main/kotlin/com/isystk/sample/web/admin/controller/html/auth/LoginForm.kt
package com.isystk.sample.web.admin.controller.html.auth

import java.io.Serializable
import javax.validation.constraints.NotEmpty

class LoginForm : Serializable {

    @NotEmpty
    var loginId: String? = null

    @NotEmpty
    var password: String? = null

    companion object {
        private const val serialVersionUID = 7593564324192730932L
    }
}